src/app_service/service/community/service.nim

```diff
@@ -179,6 +184,7 @@ QtObject:
   proc loadCommunityTags(self: Service): string
   proc loadAllCommunities(self: Service): seq[CommunityDto]
   proc asyncLoadCuratedCommunities*(self: Service)
+  proc asyncAcceptRequestToJoinCommunity*(self: Service, communityId: string, requestId: string)
```

Comment:
Would it make sense to remove legacy `acceptRequestToJoinCommunity`? It shouldn't be used anywhere, right?

Reply:
Yea, we can remove it I think. I generally am hesitant to remove APIs as they could break stuff if you aren't careful. Plus it doesn't sound off to have a sync and async option. 

Anyways, happy to remove it.